x/liquidstakeibc/keeper/msg_server.go

```diff
@@ -248,6 +249,16 @@ func (k msgServer) UpdateHostChain(
 				k.Logger(ctx).Error("Could not set withdraw address.", "chain_id", hc.ChainId)
 				return nil, fmt.Errorf("could not set withdraw address for host chain %s", hc.ChainId)
 			}
+		case KeyAutocompoundFactor:
+			autocompoundFactor, err := strconv.Atoi(update.Value)
+			if err != nil {
+				return nil, fmt.Errorf("unable to parse string to sdk.Dec")
```

Comment:
yes, error message is correct, just parse the value into Dec, instead of strconv..